share/jupyter/hub/templates/home.html

```diff
@@ -10,7 +10,7 @@
       {% endif %}
       <a id="start" class="btn btn-lg btn-success"
       {% if user.running %}
-        href="{{base_url}}user/{{user.name}}/"
+        href="{{ user.server.base_url }}"
```

Comment:
done!

On Wed, Mar 9, 2016 at 12:18 AM, Min RK <notifications@github.com> wrote:

> In share/jupyter/hub/templates/home.html
> <https://github.com/jupyter/jupyterhub/pull/472#discussion_r55484142>:
>
> > @@ -10,7 +10,7 @@
> >        {% endif %}
> >        <a id="start" class="btn btn-lg btn-success"
> >        {% if user.running %}
> > -        href="{{base_url}}user/{{user.name}}/"
> > +        href="{{ user.server.base_url }}"
>
> This should be user.url
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/jupyterhub/pull/472/files#r55484142>.
>



-- 
Yuvi Panda T
http://yuvi.in/blog